extras/systemd/foreman.socket

```diff
@@ -0,0 +1,21 @@
+[Unit]
```

Comment:
Does the socket log to the syslog? Should we set a `SyslogIdentifier=foreman-socket` on it?

Reply:
No, a socket doesn't really log anything:
```console
# journalctl -u foreman.socket
-- Logs begin at Fri 2020-03-20 11:35:32 UTC, end at Fri 2020-03-20 16:13:38 UTC. --
Mar 20 11:35:34 pipeline-upgrade-foreman-nightly-centos7.wisse.example.com systemd[1]: Listening on Puma HTTP Server Accept Sockets.
```

Now that I see this, the `Description` can be better by using Foreman instead of Puma.